Cyral/README.md

```diff
@@ -0,0 +1,7 @@
+
+# Cyral
+
+The Cyral App for Sumo Logic provides dashboards and visualizations for Cyral customers that have chosen to send their logs to the Sumo Logic platform.
+
+# Documentation
+   [Send Cyral logs to Sumo Logic](https://cyral.com/docs/integration-siem#send-cyral-logs-to-sumo-logic) 
```

Comment:
Here we should also add 
`After configuring collection you can [install](https://help.sumologic.com/05Search/Library/Apps-in-Sumo-Logic/Install-Apps-from-the-Library) the SumoLogic App from Cyril from App Catalog and use the same source category configured while creating the source`

Reply:
You can also check out documentation from other Partner Apps - https://github.com/SumoLogic/sumologic-public-partner-apps/blob/96f01c84cf4cc9d3d848bf901962392740be4eff/Tessian/README.md